Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml

```diff
@@ -61,6 +57,10 @@ configuration:
   name: proxy
   required: false
   type: 8
+- display: Incident type
```

Comment:
why did we change this?

Reply:
It was a conflict resolving issue

Reply:
lets revert, it should be located where it was

Reply:
Done